Aptoide/aptoide-client-v8
# [MOB-727] Removed ads AB test;

**What does this PR do?**

This PR removes ads AB test;

**Database changed?**

No

**How should this be manually tested?**

Make sure nothing related to ads (appview, home, search) is broken. Remember that, if wallet is installed, you should see no ads;

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-727](https://aptoide.atlassian.net/browse/MOB-727)

**Code Review Checklist**
